instrumentation/opentelemetry-instrumentation-wsgi/src/opentelemetry/instrumentation/wsgi/__init__.py

```diff
@@ -117,7 +117,14 @@ def response_hook(span: Span, environ: WSGIEnvironment, status: str, response_he
 from opentelemetry.propagators.textmap import Getter
 from opentelemetry.semconv.trace import SpanAttributes
 from opentelemetry.trace.status import Status, StatusCode
-from opentelemetry.util.http import remove_url_credentials
+from opentelemetry.util.http import (
+    OTEL_PYTHON_CAPTURE_REQUEST_HEADERS,
+    OTEL_PYTHON_CAPTURE_RESPONSE_HEADERS,
```

Comment:
Do you know if any other SIG is doing something similar `OTEL_{LANG}_CAPTURE_REQUEST_HEADERS`? I think we should be able to get consistent `OTEL_CAPTURE_REQUEST_HEADERS` accepted easily.

Reply:
I wish there was some consistency for configuring the common use-cases of instrumentations across languages.

Reply:
Changed env var for consistency with java sdk.